pkgs/top-level/all-packages.nix

```diff
@@ -5967,6 +5967,10 @@ with pkgs;
     mkFranzDerivation = callPackage ../applications/networking/instant-messengers/franz/generic.nix { };
   };
 
+  ferdium = callPackage ../applications/networking/instant-messengers/ferdium {
+    mkFranzDerivation = callPackage ../applications/networking/instant-messengers/franz/generic.nix { };
```

Comment:
Why is this not part of top-level?

Reply:
I more or less copied everything from the ferdi/franz configs (seeing as these programs are very similar) which seem to do it this way. If we make it part of top level we probably should change these packages as well?

Reply:
I concur with @SuperSandro2000. We can probably expose this to top-level and use in both `ferdium` and `franz`.

Reply:
> I more or less copied everything from the ferdi/franz configs (seeing as these programs are very similar) which seem to do it this way. If we make it part of top level we probably should change these packages as well?

yes, we don't need and shouldn't do copy pasta this all the time.